hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.test.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Item from "./Item";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+
+describe("InventoryItem", () => {
+    const image =
+        "hackathon_site/dashboard/frontend/src/assets/images/testImages/IO6e5a6.jpg";
+    const title = "Some Hardware";
+    const total = 6;
+    let currentStock = 4;
+
+    test("Has stock with no limit", () => {
+        const { getByText } = render(
+            <Item
+                image={image}
+                title={title}
+                total={total}
+                currentStock={currentStock}
+            />
+        );
+        expect(getByText("Some Hardware")).toBeInTheDocument();
+        expect(getByText(`${currentStock} OF ${total} IN STOCK`)).toBeInTheDocument();
+        expect(getByText(title)).toBeInTheDocument();
```

Comment:
LIne 22 and 24 test the same thing, but delete line 22 since its the hard coded version